packages/node-client/lib/index.ts

```diff
@@ -811,7 +811,7 @@ export class Nango {
         } else if (method?.toUpperCase() === 'PUT') {
             return this.http.put(url, config.data, options);
         } else if (method?.toUpperCase() === 'DELETE') {
-            return this.http.delete(url, options);
+            return this.http.delete(url, { ...options, data: config.data || {} });
```

Comment:
```
{ 
...options, 
...(config.data ? { data: config.data } : {})
}
```
so no data is set if no config.data, instead of passing an empty object

Reply:
Yup, thanks. Will adjust, and test tomorrow